components/list/list.html

```diff
@@ -0,0 +1,177 @@
+            <h1 class="wrapper-heading font-bold">List</h1>
+            <div class="description wrapper-description">
+                Lists are continuous, vertical indexes of text or images. They are composed of items 
+                containing primary and supplemental actions, which are represented by icons and text.
+            </div> 
+
+            <h1 class="wrapper-heading font-bold">Simple List</h1>
+            <div class="example-div">
+
+            <div class="component-showcase">
+                <div class="list-container">
+                    <ul class="list list-non-bullet">
+                      <h3 class="list-label">Options</h3>
+                        <li>Item one</li>
+                        <li>Item two</li>
+                        <li>Item three</li>
+                        <li>Item four</li>
+                        <li>Item five</li>
+                    </ul>
+                  </div> 
+           </div>
+
+           <pre>
+             <code class="code-snippet">
+
+                <div class="list-container">
+                    <ul class="list list-non-bullet">
+                      <h3 class="list-label">Options</h3>
+                        <li>Item one</li>
+                        <li>Item two</li>
+                        <li>Item three</li>
+                        <li>Item four</li>
+                        <li>Item five</li>
+                    </ul>
+                  </div> 
+            </code>
+          </pre>
+        </div>
+
+        <h1 class="wrapper-heading font-bold">Stacked List</h1>
+        <div class="description wrapper-description">
+            Stacked list have items stacked. It is mainly used to create notifications.
+        </div>
+        <div class="example-div">
+
+            <div class="component-showcase">
+                <div class="list-container">
+                    <ul class="list list-non-bullet stacked-list">
+                        <li class="stacked-list-item">Item one</li>
+                        <li class="stacked-list-item">Item two</li>
+                        <li class="stacked-list-item">Item three</li>
+                        <li class="stacked-list-item">Item four</li>
+                        <li class="stacked-list-item">Item five</li>
+                    </ul>
+                  </div> 
+           </div>
+
+           <pre>
+             <code class="code-snippet">
+
+                <div class="list-container">
+                    <ul class="list list-non-bullet stacked-list">
+                        <li class="stacked-list-item">Item one</li>
+                        <li class="stacked-list-item">Item two</li>
+                        <li class="stacked-list-item">Item three</li>
+                        <li class="stacked-list-item">Item four</li>
+                        <li class="stacked-list-item">Item five</li>
+                    </ul>
+                  </div> 
+            </code>
+          </pre>
+        </div>
+
+
+
+
```

Comment:
Very neat code with descriptive class names. 
If the extra spacing isn't intentional here, you could remove it.
The rest looks good!